tutorial/managing-software.md

```diff
@@ -0,0 +1,1394 @@
+install command again. Remember that we can install multiple packages at once
+by separating them with spaces:
+
+```bash
+sudo apt install apache2-bin=2.4.58-1ubuntu8 \
+  apache2-data=2.4.58-1ubuntu8 \
+  apache2-utils=2.4.58-1ubuntu8 \
+  apache2=2.4.58-1ubuntu8
+```
+
+In this case we're also breaking the command over multiple lines using
+backslashes (`\`) to make it easier to read, but it will still be run as a
+single command.
+
+APT will warn us that we are downgrading the package, but let us press
+<kbd>Y</kbd> to confirm (when prompted), and it will go ahead and downgrade us
+anyway. Let's run the following command again:
+
+```bash
+apt policy apache2
+```
+
+And we'll get confirmation that we're running on an older version:
+
+```bash
+apache2:
+  Installed: 2.4.58-1ubuntu8
+  Candidate: 2.4.58-1ubuntu8.4
+  Version table:
+     2.4.58-1ubuntu8.4 500
+        500 http://archive.ubuntu.com/ubuntu noble-updates/main amd64 Packages
+        500 http://security.ubuntu.com/ubuntu noble-security/main amd64 Packages
+ *** 2.4.58-1ubuntu8 500
+        500 http://archive.ubuntu.com/ubuntu noble/main amd64 Packages
+        100 /var/lib/dpkg/status
+```
+
+### APT sources
+
+You may be wondering by now "where exactly do all these packages come from?".
+We've spotted a few sources very briefly throughout this tutorial, but haven't
+paid direct attention to them yet. Let's take a little time now to define what
+we mean by all these different sources that APT can pull packages from.
+
+There is a system file that defines what we mean by all the different sources.
+We can view the file by running the following command:
+
+```bash
+cat /etc/apt/sources.list.d/ubuntu.sources
+```
+
+The APT repository is split into four **components**:
+
+|                          | Open source     | Closed source   |
+| ---                      | ---             | ---             |
+| **Officially supported** | main            | restricted      |
+| **Community supported**  | universe        | multiverse      |
+
+- **main** contains the open-source packages that are officially supported by
+  Canonical. These packages are either installed on every Ubuntu machine, or
+  are very widely used for various types of systems.
+- **universe** holds all other open-source packages in Ubuntu, which are
+  typically maintained by the Debian and Ubuntu communities, but may also
+  include additional security coverage from Canonical under
+  [Ubuntu Pro](https://ubuntu.com/pro), which is available free for personal
+  use on up to five machines.
+- **restricted** contains the packages that are officially supported by
+  Canonical but are not available under a completely free license.
+- **multiverse** contains community-maintained proprietary software -- these
+  packages are completely unsupported by Canonical.
+
+Every Ubuntu release (`noble`, `jammy`, etc) is also split into **pockets**
+which are related to the development/release lifecycle:
```

Comment:
```suggestion
Every Ubuntu release (`noble`, `jammy`, etc) is also split into **pockets** (sections of packages)
which are related to the development/release lifecycle:
```

Or similar? Your doc doesn't really note what pockets are. It defines them officially, but it doesn't clear up what they actually are to a beginner

Reply:
This is where everything has the potential to become very confusing lol. The thing that really is "sections" of packages is the components, where they're clearly divided into one or the other. You can't have a package in both main and universe afaik. Pockets aren't sections of packages, so much as sections of components. I've never been able to find a good, non-confusing explanation, so I'll try and write one.

Reply:
I've added a mermaid diagram as well, this may help a bit

Reply:
FWIW I described pockets as "sections" and components as "categories" and made a diagram here: https://ubuntu.com/landscape/docs/explanation-about-repository-mirroring. It's def still confusing, but just passing the link along if it helps to see a second opinion on how that stuff could be explained :blush: